packages/config/config/devices/0x001d/dzpd3.json

```diff
@@ -30,31 +30,44 @@
 	"paramInformation": {
 		"1": {
 			"label": "Fade On Time",
-			"description": "Time it takes to fade on",
+			"description": "Time it takes to fade on; units are seconds when value is 1-127, and minutes (subtracting 127) when value is 128-253.",
```

Comment:
Sorry, this is actually info we're trying to remove form descriptions. We don't describe ranges usually. And the unit can in a unit field. I don't think you need a description at all here.

Reply:
I think you suggested otherwise above, and, without a description or units, it's impossible to understand:
![image](https://user-images.githubusercontent.com/8227920/109584250-302ce680-7acf-11eb-95dd-6a2ffd93878d.png)

Reply:
Ah, disregard. Missed the changing unit. Go back to how you had it. Apologies.

Reply:
Funny how they sometimes come so close to using a spec-defined encoding, but fail on the last second:
![grafik](https://user-images.githubusercontent.com/17641229/109636005-5728fe00-7b4b-11eb-9d8d-3b22906aa5b6.png)